tokens/color/base.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
this is super freaking exciting!! Thanks for exploring this! 😀  🎉

Reply:
cc @dcaryll @coreyvickery

Reply:
I used the hex that is defined in the system that but would it make the colors more globally useable if they were represented in rgb?

Reply:
i think hex is totally fine for now. if we have a good reason to try to convert them later we can